docs/patterns/priority-queue.md

```diff
@@ -87,74 +87,47 @@ In the figure above, the application creates several messages and assigns a cust
-
-The `Run` method in the `PriorityWorkerRole` class arranges for the virtual `ProcessMessage` method (also defined in the `PriorityWorkerRole` class) to be run for each message received on the queue. The following code shows the `Run` and `ProcessMessage` methods. The `QueueManager` class, defined in the PriorityQueue.Shared project, provides helper methods for using Azure Service Bus queues.
+The PriorityQueue solution available on [GitHub](https://github.com/mspnp/cloud-design-patterns/tree/master/priority-queue) contains an implementation of this approach. This solution contains Azure Function projects named `PriorityQueueConsumerHigh` and `PriorityQueueConsumerLow`. These Azure Functions integrate with Azure Service Bus via triggers and bindings, they connect to different subscriptions defined in the ServiceBusTrigger and react to the incoming messages.
 
 ```csharp
-public class PriorityWorkerRole : RoleEntryPoint
-{
-  private QueueManager queueManager;
-  ...
-
-  public override void Run()
-  {
-    // Start listening for messages on the subscription.
-    var subscriptionName = CloudConfigurationManager.GetSetting("SubscriptionName");
-    this.queueManager.ReceiveMessages(subscriptionName, this.ProcessMessage);
-    ...;
-  }
-  ...
-
-  protected virtual async Task ProcessMessage(BrokeredMessage message)
-  {
-    // Simulating processing.
-    await Task.Delay(TimeSpan.FromSeconds(2));
-  }
-}
+public static class PriorityQueueConsumerHighFn
+    {
+        [FunctionName("HighPriorityQueueConsumerFunction")]
+        public static void Run([ServiceBusTrigger("messages", "highPriority", Connection = "ServiceBusConnection")]string highPriorityMessage, ILogger log)
+        {
+            log.LogInformation($"C# ServiceBus topic trigger function processed message: {highPriorityMessage}");
+        }
+    }
 ```
 
-The `PriorityQueue.High` and `PriorityQueue.Low` worker roles both override the default functionality of the `ProcessMessage` method. The code below shows the `ProcessMessage` method for the `PriorityQueue.High` worker role.
-
-```csharp
-protected override async Task ProcessMessage(BrokeredMessage message)
-{
-  // Simulate message processing for High priority messages.
-  await base.ProcessMessage(message);
-  Trace.TraceInformation("High priority message processed by " +
-    RoleEnvironment.CurrentRoleInstance.Id + " MessageId: " + message.MessageId);
-}
-```
+An administrator can configure how many instances the functions on the app service consumption can scale out to by setting a value to the WEBSITE_MAX_DYNAMIC_APPLICATION_SCALE_OUT application setting. Typically there'll be more instances of the `PriorityQueueConsumerHigh` function than the `PriorityQueueConsumerLow` function.
```

Comment:
```suggestion
An administrator can configure how many instances the functions on the app service consumption can scale out to by setting a value to the [`WEBSITE_MAX_DYNAMIC_APPLICATION_SCALE_OUT`](https://docs.microsoft.com/azure/azure-functions/functions-app-settings#website_max_dynamic_application_scale_out) application setting. Typically, there'll be more instances of the `PriorityQueueConsumerHigh` function than the `PriorityQueueConsumerLow` function.
```

Reply:
But more importantly, this setting is deprecated.  Please update to use: https://docs.microsoft.com/azure/azure-functions/event-driven-scaling#limit-scale-out

Reply:
@magrande -- just wanted to make sure you saw this above.

Reply:
@ckittel I will take care of this in the samples repo, thanks !

Reply:
Thanks. The new wording looks good.  I'll move this PR over to the internal repo, and will close this one out when complete.